.github/workflows/reviewdog.yml

```diff
@@ -0,0 +1,28 @@
+name: Automated Code Reviews
+on: [pull_request]
+
+permissions:
+  contents: read
+  pull-requests: write
+
+jobs:
+  bearer:
+    name: Bearer Security Analysis
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: reviewdog/action-setup@v1
+        with:
+          reviewdog_version: latest
+      - name: Bearer
+        uses: bearer/bearer-action@v2
+        with:
+          diff: true
+          format: rdjson
```

Comment:
@that-jill I have no idea about this, but is rdjson & rd.json 2 separate things or a typo?